[JENKINS-63938] Add storage of commentBody in GitHubPullRequestCommentCause

jebentier opened the pull request:
This PR is to address the first part of the issue presented by #22 by adding the storage of the GitHub
comment body to the resulting build cause. With this, one can now easily retreive the comment body
that triggered the build to start so that it can be parsed during the build.  Further work can
be done on top of this PR to store this information as an environment variable on the Jenkins job
itself, but this alone helps to remove the need to create an API call out to GitHub for information
that was already sent in the initial webhook.

bluesliverx commented on src/main/java/com/adobe/jenkins/github_pr_comment_build/GitHubPullRequestCommentCause.java:
```diff
@@ -7,13 +7,25 @@
      */
     public GitHubPullRequestCommentCause(String commentUrl) {
         this.commentUrl = commentUrl;
+        this.commentBody = "Unknown";
```
I feel that this is not correct. it really should be an empty string. Otherwise how could someone tell the difference between "Unknown" as a comment and this default case?

bluesliverx replied:
Alternatively, you could take out this constructor since I believe it's not used anymore, although people outside of the plugin may have depended on it. However, they can also just update their usage if they are using internal classes, so I think I'm leaning towards just removing this version of the constructor.

jebentier replied:
Thank you for getting the build working again. I've gone ahead and removed the old constructor. I wasn't sure at first how backwards compatible things needed to be, but agree that it's completely unnecessary now internally.

bluesliverx commented:
Ok, finally got the build working again in #28, but looking at the PR a little more in depth I noticed something and commented on it. Please get that fixed and we can get this merged in and released.

jebentier commented:
@bluesliverx Thank you for getting to this, and for getting the build fixed. I have gone ahead and rebased off your recent changes that are in master and addressed your comment.
